internal/compiler/utils.go

```diff
@@ -16,12 +19,35 @@ const (
 	AuthorizationPolicySchema SchemaFile = "authorizationPolicy.json"
 )
 
-var schemaDefinitions = map[SchemaFile]interface{}{}
+var schemaDefinitions = map[SchemaFile]any{}
+
+var loadOnce = sync.OnceValue(func() error {
```

Comment:
This is a bit confusing. `sync.OnceValue` is for retrieving the value, but what we care about isn't `error`, it's the functions side-effect. And if it actually does error, we'll panic. So I think I'd be less confused about [`sync.OnceFunc`](https://pkg.go.dev/sync#OnceFunc) and calls `panic()` in that func...🤔 

Overall, I like the lazy init here. It's nice to get rid of that work on init.

Reply:
I went with this pattern as I liked having the caller decide the meaning of an error. There's only one caller now so it doesn't really matter, but still :)